src/Android/Services/CryptoPrimitiveService.cs

```diff
@@ -33,5 +35,20 @@ public byte[] Pbkdf2(byte[] password, byte[] salt, CryptoHashAlgorithm algorithm
             generator.Init(password, salt, iterations);
             return ((KeyParameter)generator.GenerateDerivedMacParameters(keySize)).GetKey();
         }
+
+        public byte[] Argon2id(byte[] password, byte[] salt, int iterations, int memory, int parallelism)
+        {
+            JavaSystem.LoadLibrary("argon2");
+            // TODO: Do we need to pass this in somehow like PBKDF2, based on an algorithm?
+            int keySize = 32;
+            var key = new byte[keySize];
```

Comment:
Is the `keySize` going to be configured based on a parameter or will it stay as is for this PR?

Reply:
Removed comment. It should just be 32 bit key for now.